playwright-e2e/tests/rgp/mailing-list-signup.spec.ts

```diff
@@ -0,0 +1,68 @@
+import {expect} from '@playwright/test';
```

Comment:
`rgp/mailing-list-signup.spec` have been renamed and moved. no longer exists.

Reply:
removed!